lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -70,9 +70,8 @@ static void DCE(Function *F) {
 
   // Remove unused nodes. Do not remove unused vars because they are the
   // interface to the user program.
-  bool changedLocally = true;
-  do {
-    changedLocally = false;
+  while(true) {
```

Comment:
nit: I think you forgot to `clang-format`, as I believe there should be a space after `while` 😃

Reply:
I believe you're right :)

Reply:
Done.